navapbc/wic-mt-demo-project-eligibility-screener
# [Post-Demo] Refactor the infrastructure Terraform

## Ticket

https://wicmtdp.atlassian.net/browse/{TICKET_ID}

## Changes
* created `network.tf` 
* add additional config to the ecr repo

## Context for reviewers
> After completing [PRP-116: Migrate terraform for Eligibility Screener out of project repo and into Eligibility Screener repoREADY FOR REVIEW](https://wicmtdp.atlassian.net/browse/PRP-116) , we should refactor the terraform for the Eligibility Screener to use the same structure that we adopted for the PRP, based on the template repo.

## Testing
When this is out of `draft`:
1. Run `terraform plan`
2. Plan should run successfully with no errors
3. Run `terraform apply`
4. No changes should be committed since all of the listed resources currently exist
